src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+  width: fit-content;
+
+  &__star {
+    margin-right: 4px;
+    height: 16px;
+    width: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &--4 &__star:nth-child(-n + 4) {
```

Comment:
The selector `&--4 &__star:nth-child(-n + 4)` is correct in terms of targeting the first four stars, but ensure that the modifier `--4` is used consistently throughout your HTML to apply this style correctly. This is more of a reminder to check the HTML structure.